fhir-persistence-jdbc/src/test/java/com/ibm/fhir/persistence/jdbc/test/util/DerbyInitializer.java

```diff
@@ -116,14 +113,7 @@ public void bootstrapDb(boolean reset) throws FHIRPersistenceDBConnectException,
         }
         else {
             System.out.println("Bootstrapping database");
-            final String url = DERBY_TRANSLATOR.getUrl(dbProps);
-            try (Connection connection = DriverManager.getConnection(url + ";create=true")) {
```

Comment:
removing this makes create=false.  I think in this case don't you want to set to true? I can't think of the exact reason we hit this before, but just worth checking that create=true is specified in DerbyFhirDatabase.

Reply:
`new DerbyFhirDatabase()` is how Robin was instantiating the database R4JDBCExamplesTest, so I was just trying to be consistent with that.